src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+    LOSE(0, 0),
+    FIFTH(3, 5000),
```

Comment:
https://countryxide.tistory.com/52
```suggestion
    FIFTH(3, 5_000),
```